src/libpriv/rpmostree-rojig-assembler.c

```diff
@@ -172,7 +172,7 @@ peel_entry_pathname (struct archive_entry *entry,
                      GError    **error)
 {
   const char *pathname = archive_entry_pathname (entry);
-  static const char prefix[] = "./usr/lib/ostree-jigdo/";
+  static const char prefix[] = "./usr/lib/ostree-rojig/";
```

Comment:
This is a breaking change -- we could support the old one too, I suppose? Though since it's still experimental, it seems fine to just break it.

Reply:
Urgh.  I think what happened is I hit https://ci.centos.org/view/Atomic/job/fahc-treecompose/11570/console and then changing it fixed it...but the real problem was likely inconsistency in the spec file, not the naming.

So...hm.  I think my vote is to fix the fedora-atomic spec files.  Sorry about doing that without peer review.